constants/constants.py

```diff
@@ -19,25 +20,26 @@
     'conditional': ['('],
     'function': ['('],
     'mainuwu': ['-'],
-    'int_float': [' ', *ATOMS['arithmetic_operator'], ')', '}', ']', '~', '!', r'&', '|', '>', '<', '='],
+    'int_float': [' ', *ATOMS['general_operator'], ')', '}', ']', '~', '!', r'&', '|', '>', '<', '='],
     'string': [' ', ')', ',', '&', '}', '[', ']', '~', '!', '='],
-    'id': [' ', '~', ',', ')', '[', ']', '}', *ATOMS['arithmetic_operator'], '!', r'&', '|', '>', '<', '=', '.',],
     'assign_delim': [*ATOMS['alpha'], *ATOMS['number'], '{', ' ', '-' '('],
-    'arithmetic_operator': [*ATOMS['alpha_small'], *ATOMS['number'], ' ', '-', '('],
-    'logical_delim': ['"', *ATOMS['alpha_small'], *ATOMS['number'], ' ', '-', '('],
-    'open_brace': [*ATOMS['number'], ' ', '"', *ATOMS['alpha_small'], r'\n', '>'],
+    'operator_delim': [*ATOMS['alpha'], *ATOMS['number'], ' ', '-', '('],
+    'logical_delim': ['"', *ATOMS['alpha'], *ATOMS['number'], ' ', '-', '('],
+    'open_brace': [*ATOMS['number'], ' ', '"', *ATOMS['alpha'], r'\n', '>'],
     'close_brace': ['~', ' ', ',', ')', '>'],
-    'open_parenthesis': [*ATOMS['number'], *ATOMS['alpha_small'], ' ', '-', r'\n', '>'],
-    'close_parenthesis': [' ', *ATOMS['arithmetic_operator'], r'\n', '~', '>', '.'],
-    'open_bracket': [*ATOMS['number'], '-'],
-    'double_open_bracket': [' ', r'\n', *ATOMS['alpha_small'], '>'],
-    'close_bracket': [' ', '~', ',', ')', '[', ']', '}', *ATOMS['arithmetic_operator'], '!', r'&', '|', '>', '<', '=', '.',],
-    'double_close_bracket': [' ', r'\n', *ATOMS['alpha_small'], '>'],
-    'unary': ['~', ')', '', *ATOMS['arithmetic_operator']],
-    'concat': [' ', '"', *ATOMS['alpha_small']],
-    'line': [None],
-    'comma': [*ATOMS['alpha_small'], ' ', *ATOMS['number'], '"', '-', r'\n', '>'],
-    'dot_op': [*ATOMS['alpha_small']],
+    'open_parenthesis': [*ATOMS['number'], *ATOMS['alpha'], ' ', '-', r'\n', '>'],
+    'id': [' ', '~', ',', ')', '[', ']', '}', *ATOMS['general_operator'], r'&', '|', '.'],
+    'close_parenthesis': [' ', *ATOMS['general_operator'], '&', '|', r'\n', '~', '>', '.', ',', ')', '(', '[', ']', '}'],
+    'open_bracket': [*ATOMS['number'], '-', *ATOMS['alpha']],
+    'double_open_bracket': [' ', r'\n', *ATOMS['alpha'], '>'],
+    'close_bracket': [' ', '~', ',', ')', '[', ']', '}', *ATOMS['general_operator'], r'&', '|', '.',],
+    'double_close_bracket': [' ', r'\n', *ATOMS['alpha'], '>'],
+    'unary': ['~', ')', *ATOMS['general_operator']],
+    'concat': [' ', '"', *ATOMS['alpha']],
+    'line': [r'\n', ' ', *ATOMS['alpha']],
+    'comma': [*ATOMS['alpha'], ' ', *ATOMS['number'], '"', '-', r'\n', '>'],
+    'dot_op': [*ATOMS['alpha'], '[', '('],
     'start_done': [r'\n', ' ', '>'],
     'nuww': [' ', '~', ')', '}', ','],
+    'whitespace': [' ']
```

Comment:
what's this for?

Reply:
fwunc and cwass keywords, which is also what's on our TD